proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,140 @@
+
+==============
+OCDEP: Division Identifier Governance
+==============
+
+:Created: 2017-10-26
+:Author: Donny Bridges
+:Status: Proposed
+
+Overview
+========
+
+A formalized governance model for the creation, acceptance, and upkeep of Open Civic Data Division Identifiers (OCDIDs).
+
+Rationale
+=========
+
+Recently, a number of civic data organizations, tech companies, and US state governments have expressed interest in taking a greater role in the creation, upkeep, and wider adoption of OCDIDs. Currently, the governance structure for division IDs  as defined in OCDEP2 is "informal... led by the project’s early contributors and informed by the Open Civic Data Google Group." Increased adoption and interest makes this informality untenable for reasons including:
+
+* Bottlenecks arising when large amounts of new identifiers are proposed but only a few people have permission to review and approve pull requests.
+* Subject-matter experts and organizations that have extensive experience working with OCDIDs are willing to take greater responsibility, but currently do not have a formal path for doing so.
+* Government entities and other local experts who may wish to contribute to the OCDID repository do not have a clear idea of how the process for contribution works, making it more difficult for them to justify contributing.
+
+We propose to formalize this structure in order to relieve some of the burden on those currently responsible for maintaining the repository, as well as allowing those whose work depends on OCDIDs to have more direct control on their upkeep.
+
+
+Implementation
+==============
+
+Roles & Responsibilities
+------------------------
+``User``
+~~~~~~~~
+Anyone using OCDIDs may contribute through group discussion, reporting issues, or assisting with subject-matter expertise.
+The agreed upon communication channels for each country's community will be publicly displayed in the Open Civic Data documentation.
```

Comment:
Just wanted to note that it is important for us to encourage user participation by making the adoption bar low for the technology to be used for group discussion/communication. Specifically if a user need to be on a specific platform that requires additional account creation, that may not be ideal to try to include more people who can participate on the project, that said, participating using an existing email address might be the minimum requirement.